spec/classes/qpid_client_spec.rb

```diff
@@ -4,7 +4,7 @@
   on_supported_os.each do |os, facts|
     context "on #{os}" do
       let :facts do
-        facts
+        facts.merge(:systemd => false)
```

Comment:
In which case do we run without systemd? el7 is the only supported OS. Maybe we should set a custom fact to always true in the spec helper so we don't need to override it.